packages/components/src/theme/charts.scss

```diff
@@ -180,3 +183,68 @@
 .r-report__files .attachment-card:hover {
     text-decoration: none;
 }
+
+.chart-builder-modal {
```

Comment:
This block of styles being nested means we're created *very* specifically scoped styles, and I am unsure that we actually need to do that. In general we really only should be nesting styles if we want the styles scoped very specifically (and IME we don't want that for most cases).

Reply:
[Here's a blog post ](https://bradfrost.com/blog/post/sass-selectors-to-nest-or-not-to-nest/)that goes over to nest or not nest, and included a (non-scientific) survey of what people were doing, and the general consensus was "We started nesting, but over time we found out it was not worth it, and have since stopped". I agree with the consensus here, in particular I find it hard to find the styles I care about when looking at DOM elements (which is something I do often) if the styles are nested.